include/libtorrent/add_torrent_params.hpp

```diff
@@ -297,6 +297,10 @@ namespace libtorrent
 		// url seeds to be added to the torrent (`BEP 17`_).
 		std::vector<std::string> url_seeds;
 
+		// peers to add to the torrent, to be tried to be connected to as
+		// bittorrent peers.
+		std::vector<tcp::endpoint> peers;
```

Comment:
I was afraid of this, but, is there a way to add something without breaking ABI?